### PR TITLE
Met AttributeError due to screenshot returns None

### PR DESCRIPTION
**Pull request by WeizhongX**
Stack trace as belw:
File "/b/s/w/ir/third_party/wpt_tools/wpt/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 405, in run_func
    self.result = True, self.func(self.protocol, self.url, self.timeout)
File "/b/s/w/ir/third_party/wpt_tools/wpt/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 620, in _screenshot
    if screenshot.startswith("data:image/png;base64,"):
AttributeError: 'NoneType' object has no attribute 'startswith'

### CODE REVIEW

File: tools/wptrunner/wptrunner/executors/executorwebdriver.py [hunk 1]
```diff
@@ -630,6 +630,8 @@ def _screenshot(self, protocol, url, timeout):
         self.protocol.base.execute_script(self.wait_script, True)
 
         screenshot = self.protocol.webdriver.screenshot()
+        if screenshot is None:
+            raise error.UnknownErrorException()
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/executors/executorwebdriver.py:634]: It seems odd to return a webdriver error here, given that there apparently wasn't a protocol-level error, but we somehow got a `None` result (maybe the browser crashed?). Can we at least just raise a normal Python exception like `ValueError`?

**Review by jgraham — CHANGES_REQUESTED** [review]

File: tools/wptrunner/wptrunner/executors/executorwebdriver.py (hunk 1 above)
Comment (WeizhongX) [reply tools/wptrunner/wptrunner/executors/executorwebdriver.py]: updated, thanks!

**Review by jgraham — APPROVED** [review]

File: tools/wptrunner/wptrunner/executors/executorwebdriver.py [hunk 2]
```diff
@@ -630,6 +630,8 @@ def _screenshot(self, protocol, url, timeout):
         self.protocol.base.execute_script(self.wait_script, True)
 
         screenshot = self.protocol.webdriver.screenshot()
+        if screenshot is None:
+            raise ValueErrori('screenshot is None')
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/executors/executorwebdriver.py:634]: You'll need to fix the typo to make CI pass :)

Comment (WeizhongX) [reply tools/wptrunner/wptrunner/executors/executorwebdriver.py]: Thanks

**Pull request merged**